zone/raids.cpp

```diff
@@ -2278,13 +2282,18 @@ void Raid::SendRaidNotes()
 	VerifyRaid();
 
 	for (const auto& c : GetMembersWithNotes()) {
-		auto outapp = new EQApplicationPacket(OP_RaidUpdate, sizeof(RaidGeneral_Struct));
-		auto note = (RaidGeneral_Struct*)outapp->pBuffer;
-		note->action = raidSetNote;
-		strn0cpy(note->leader_name, c.member_name, 64);
-		strn0cpy(note->player_name, GetLeaderName().c_str(), 64);
-		strn0cpy(note->note, c.note, 64);
+		RaidNote_Struct temp;
+		temp.general.action = raidSetNote;
+		temp.general.parameter = 0;
+		temp.general.unknown1 = 0;
+		strn0cpy(temp.general.leader_name, c.member_name, 64);
+		strn0cpy(temp.general.player_name, GetLeaderName().c_str(), 64);
+		temp.note = c.note;
+
+		auto outapp = new EQApplicationPacket(OP_RaidUpdate, sizeof(RaidNote_Struct));
+		outapp->pBuffer = (unsigned char*)&temp;
 		QueuePacket(outapp);
+		outapp->pBuffer = nullptr;
```

Comment:
Same as above with the `outapp` buffer leak, can just cast the buffer to `RaidNote_Struct` and modify directly